index.html

```diff
@@ -1,84 +1,305 @@
+        <button id="sidebar-toggle"
+          class="text-sm font-medium ring-offset-background transition-colors focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:pointer-events-none disabled:opacity-50 rounded-none border-l border-gray-300 justify-center items-center flex h-10 w-10 btn-theme-dark">
+          <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24"
+            viewBox="0 0 24 24" fill="none" stroke="currentColor"
+            stroke-width="2" stroke-linecap="round" stroke-linejoin="round"
+            class="h-6 w-6 transition-transform"><polyline
+              points="15 18 9 12 15 6"></polyline></svg></button>
       </div>
     </div>
+    <!-- MAP -->
     <div id='map' style='width: 100wh; height: 100vh'></div>
-    <!-- END -->
+    <!-- MAP SETTINGS BUTTON -->
+    <div class="flex flex-col space-y-2 absolute right-6 top-6 z-40">
+      <button
+        id="map-settings-button"
+        class="relative flex h-8 w-8 items-center p-1.5 transition-colors border shadow-sm active:outline active:outline-2 btn-theme-dark"
+        aria-label="Map settings" type="button" data-state="open"
+        aria-haspopup="dialog" aria-expanded="true">
+        <!-- Map settings icons can not be imported as it has to change color -->
+        <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24"
+          viewBox="0 0 24 24" fill="none" stroke="currentColor"
+          stroke-width="2" stroke-linecap="round" stroke-linejoin="round"
+          class="h-full w-full">
+          <path
+            d="M12.22 2h-.44a2 2 0 0 0-2 2v.18a2 2 0 0 1-1 1.73l-.43.25a2 2 0 0 1-2 0l-.15-.08a2 2 0 0 0-2.73.73l-.22.38a2 2 0 0 0 .73 2.73l.15.1a2 2 0 0 1 1 1.72v.51a2 2 0 0 1-1 1.74l-.15.09a2 2 0 0 0-.73 2.73l.22.38a2 2 0 0 0 2.73.73l.15-.08a2 2 0 0 1 2 0l.43.25a2 2 0 0 1 1 1.73V20a2 2 0 0 0 2 2h.44a2 2 0 0 0 2-2v-.18a2 2 0 0 1 1-1.73l.43-.25a2 2 0 0 1 2 0l.15.08a2 2 0 0 0 2.73-.73l.22-.39a2 2 0 0 0-.73-2.73l-.15-.08a2 2 0 0 1-1-1.74v-.5a2 2 0 0 1 1-1.74l.15-.09a2 2 0 0 0 .73-2.73l-.22-.38a2 2 0 0 0-2.73-.73l-.15.08a2 2 0 0 1-2 0l-.43-.25a2 2 0 0 1-1-1.73V4a2 2 0 0 0-2-2z">
+          </path>
+          <circle cx="12" cy="12" r="3"></circle>
+        </svg>
+      </button>
+    </div>
+
+    <!-- MAP SETTINGS POPUP -->
+    <div
+      class="fixed z-40 pointer-events-none flex items-center justify-center right-6 top-11 w-[370px] h-[280px] opacity-0"
```

Comment:
Once you change the button size, you'll need to change the right position.
```suggestion
      class="fixed z-40 pointer-events-none flex items-center justify-center right-7 top-11 w-[370px] h-[280px] opacity-0"
```